mysql-test/suite/rpl/t/rpl_stop_slave_error.test

```diff
@@ -6,11 +6,26 @@ source include/master-slave.inc;
 
 connection master;
 sync_slave_with_master;
+
+--let $iter=1000
+--echo # MDEV-32892: Repeatedly starting/stopping io_thread..
+--disable_query_log
+while ($iter)
+{
+  stop slave io_thread;
+  start slave io_thread;
+  --dec $iter
+}
+--enable_query_log
```

Comment:
Can we use something less than 1000 iterations here? I can imagine it was useful during development testing, but I think it will take a long time to run this part of the test. So I'd reduce this a lot before pushing, say so that it doesn't take more than a couple seconds to run - any later regression in the code should still be caught in the buildbot sooner or later.

Reply:
On my laptop, 1000 iterations takes less than a second (`0.8s`), but sure, I can reduce it just in case, say for  overwhelmed BB runs.